src/components/package-port/CodeAndPreview.tsx

```diff
@@ -141,13 +128,33 @@ export function CodeAndPreview({ pkg }: Props) {
     usePackageFilesLoader(pkg)
 
   useEffect(() => {
+    setState((prev) => ({
+      ...prev,
+      pkg: pkg,
+    }))
+
+    if (!pkg) {
+      setState((prev) => ({
+        ...prev,
+        pkgFilesWithContent: [{ path: "index.tsx", content: defaultCode }],
+        initialFilesLoad: [{ path: "index.tsx", content: defaultCode }],
+        currentFile: "index.tsx",
+      }))
+    } else {
+      setState((prev) => ({
+        ...prev,
+        currentFile: "",
```

Comment:
empty strings are a huge antipattern when they represent a default state. This caused some huge issues in core in the past!!

```suggestion
        currentFile: null,
```